internal/signerverifier/ssh/ssh.go

```diff
@@ -56,10 +56,19 @@ func (v *Verifier) Public() crypto.PublicKey {
 	return v.sshKey.(ssh.CryptoPublicKey).CryptoPublicKey()
 }
 
+func (v *Verifier) MetadataKey() *sv.SSLibKey {
+	return &sv.SSLibKey{
+		KeyID:   v.keyID,
+		KeyType: SSHKeyType,
+		Scheme:  v.sshKey.Type(),
+		KeyVal:  sv.KeyVal{Public: base64.StdEncoding.EncodeToString(v.sshKey.Marshal())},
+	}
```

Comment:
IMO it would be a good idea to factor this into a helper to make sure that `MetadataKey` and `NewKeyFromFile` return the same Key.  The helper can have an optional keyid argument, to pass `v.KeyID` when called here, or compute a default, when called from `NewKeyFromFile`.